### PR TITLE
feat(xdsl): add email pro

### PR DESCRIPTION
**Pull request by cbourgois**
Add services for `/pack/xdsl/:packName/emailPro` and `/xdsl/email/pro/:email` endpoints.

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#519.1 ✘  
* build
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build#520.0 ✘  
* build
  * build ✔ 
  * test ✘

File: src/pack/xdsl/emailPro/pack-xdsl-email-pro.v6.service.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+
```
Comment (antleblanc) [inline src/pack/xdsl/emailPro/pack-xdsl-email-pro.v6.service.js]: useless empty line.

Comment (cbourgois) [reply src/pack/xdsl/emailPro/pack-xdsl-email-pro.v6.service.js]: Fixed.

**Pull request merged**